### PR TITLE
Update CSP to include /static/ path for frontend files

### LINKED ISSUE #10220: Update CSP once we use a sub-folder
After https://github.com/mozilla/addons-frontend/issues/10219 and is verified, we should update the CSP config.

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #10220

### CODE REVIEW

File: tests/unit/amo/middleware/test_cspMiddleware.js [hunk 1]
```diff
@@ -39,11 +39,12 @@ describe(__filename, () => {
         // eslint-disable-next-line global-require
         const config = require('config');
         const cspConfig = config.get('CSP').directives;
-        expect(cspConfig.scriptSrc).toContain(cdnHost);
```
Comment (muffinresearch) [inline tests/unit/amo/middleware/test_cspMiddleware.js:42]: Some of these could be inverted (with a comment), in-case someone looks to 'optimize' the CSP back to a single origin.

**Review by muffinresearch — APPROVED** [review]
r+wc

File: tests/unit/amo/middleware/test_cspMiddleware.js (hunk 1 above)
Comment (willdurand) [reply tests/unit/amo/middleware/test_cspMiddleware.js:42]: What should be inverted?

Comment (muffinresearch) [reply tests/unit/amo/middleware/test_cspMiddleware.js:42]: The test expectation. To cover disallowing an origin without a path.

**Pull request merged**